skimage/feature/match.py

```diff
@@ -59,7 +59,11 @@ def match_descriptors(descriptors1, descriptors2, metric=None, p=2,
         else:
             metric = 'euclidean'
 
-    distances = cdist(descriptors1, descriptors2, metric=metric, p=p)
+    kwargs = {}
```

Comment:
@hmaarrfk would you mind fixing the docstring as well? Several missing `, optional` over here...

Reply:
ok.